using/faqs.md

```diff
@@ -80,53 +90,60 @@ If you are receiving permission denied errors when trying to upgrade the command
+If so, use the tools provided by your system to change the permissions of the directory to grant write access to your user and trying upgrading again.
+If you are not sure if you created the directory, or the returned path is a system path, please use your system tools to uninstall the command-line client and reinstall using the [interactive walkthrough][interactive-walkthrough].
 
 ## I get the "16-bit MS-DOS Subsystem" error dialog after upgrading on Windows. What should I do?
 
-Prior to version 3.0.5 of the Exercism command-line client, there was a bug in the upgrade command that would replace the command-line client binary file with a single text file causing the "16-bit MS-DOS Subsystem" error. To resolve this issue remove the corrupt binary and reinstall the command-line client using the [interactive walkthrough](https://exercism.org/cli-walkthrough).
+Prior to version 3.0.5 of the Exercism command-line client, there was a bug in the upgrade command that would replace the command-line client binary file with a single text file causing the "16-bit MS-DOS Subsystem" error.
+To resolve this issue remove the corrupt binary and reinstall the command-line client using the [interactive walkthrough][interactive-walkthrough].
 
 ## Mentored Mode
 
 ## My queue hasn't updated in a while - is there something wrong?
 
-While wait times can be longer than normal for a number of reasons (number of mentors, time of year), it can also appear that the queue is not updating. This is not unusual and you should see movement fairly soon. If you your wait is far longer than the average, refer to [Opening an Issue](#opening-an-issue) below.
+While wait times can be longer than normal for a number of reasons (number of mentors, time of year), it can also appear that the queue is not updating.
+This is not unusual and you should see movement fairly soon.
+If you your wait is far longer than the average, refer to [Opening an Issue][opening-an-issue] below.
 
 ## How many mentors does my track have? I only see a handful on the website.
 
-The website only shows mentors who have provided bio information for the website, not the actual number of mentors who are actively reviewing solutions. Rest assured, there are mentors working through their queues so hang in there!
+The website only shows mentors who have provided bio information for the website, not the actual number of mentors who are actively reviewing solutions.
+Rest assured, there are mentors working through their queues so hang in there!
 
 ## I submitted the wrong solution / file. What can I do?
 
-There isn't a way to delete submissions so just submit the right file and it will show up as a new iteration in the queue. Feel free to leave a note for your mentor with the original submission clarifying what happened.
+There isn't a way to delete submissions so just submit the right file and it will show up as a new iteration in the queue.
```

Comment:
This is no longer true:
* On the exercise page in question, navigate to "Your Iterations". If the unwanted iteration is folded, click the circled arrow. In the "dots" menu, select "Delete iteration".